### PR TITLE
add examples for Local models

### PR DESCRIPTION
**Pull request by VinciGit00**

### CODE REVIEW

**Comment by PeriniM** [comment]
it doesn't make sense to add a docker folder, only the base_url needs to be changed

File: examples/local_models/Ollama/result.json
Comment (PeriniM) [inline examples/local_models/Ollama/result.json:1]: file to delete

Comment (VinciGit00) [reply examples/local_models/Ollama/result.json:1]: deleted all

File: examples/local_models/Ollama/result.csv
[the remark this answers is not in the document]
Comment (VinciGit00) [reply examples/local_models/Ollama/result.csv:1]: deleted all

**Comment by VinciGit00** [comment]
> it doesn't make sense to add a docker folder, only the base_url needs to be changed

if someone does not know the difference it could be a problem, it is better to split it, in this way they are ready to go

**Pull request merged**